deploy/README.md

```diff
@@ -1,31 +1,180 @@
+The Arrebol Service manager the job execution in the worker nodes and monitor the job state. It mean that Arrebol uses the Worker Node resources to run the jobs.
+
+## Infrastruture
+
+You just need a machine for Arrebol Service.
+
+### Requirements
+* vCPU: 2
+* RAM: 2GB
+* FREE DISK SPACE: 10GB
+* OS: Ubuntu 16 or higher 
+
+### Security Group
+* Custom TCP Rule to allow ingress in the port **8080** to access **Arrebol API**
+* Custom TCP Rule to allow ingress in the port **5432** to access **Postgresql DB** (Optional)
+* Custom TCP Rule to allow ingress in the port **15432** to access **Pgadmin** (Optional)
+
+## Setup
+
+### 1.Download the repository
+
+Log in the arrebol host and follow the instructions.
+
+To download thre repository is need the git installed. Case it was not installed, run theses commands to install:
+```bash
+sudo apt update
+sudo apt install -y git
+```
+Run the below commands to download the arrebol repository:
+```bash
+git clone -b feature/remote-worker https://github.com/ufcg-lsd/arrebol.git
+```
+
+### 2.Install dependencies
+
+Log in the coordination host and run the following commands to install dependencies.
 
   ```
+  cd arrebol/deploy
   sudo bash setup.sh
   ```
 
-## Configuration
+### 3. Fill configuration files
+
+Go to the _deploy/config_ directory  inside _arrebol_:
+```bash
+cd arrebol/deploy/config
+```
+
+It was the configuration files. It is necessary to edit the files in order: **postgres.env**, **pgadmin.env**, **application.properties** and **arrebol.json**.
 
-All the configuration files are within the `deploy/config` directory. It is necessary to edit:
+#### Postgres Configuration
 
-* The `postgres.env`, to define a password to the database;
-* The `pgadmin.env`, to define an the database admin credentials;
-* Assign the previously defined database password in the `spring.datasource.password` property on `application.properties` file;
-* Configure the `arrebol.json` file to tune Arrebol internals. You will write the worker nodes ip there.
+File: postgres.env
+
+```
+POSTGRES_PASSWORD=
+```
 
-## Install
+The **POSTGRES_PASSWORD** define a password to the postgres database. The postgres user is **postgres** by default. 
 
-After the configuration, execute the `deploy/deploy-stack.sh` script to install and run the Arrebol server.
+Considering that password is **@rrebol**. The content of the postgres.env file would be:
 
-  ```
-  sudo bash deploy-stack.sh
-  ```
+```
+POSTGRES_PASSWORD=@arrebol
+```
+
+#### PgAdmin Configuration
+
+File: pgadmin.env
+
+```
+PGADMIN_DEFAULT_EMAIL=
+PGADMIN_DEFAULT_PASSWORD=
+```
+
+The **PGADMIN_DEFAULT_EMAIL** and **PGADMIN_DEFAULT_PASSWORD** define a admin user to login in the pgadmin. 
+The **PGADMIN_DEFAULT_EMAIL** does not necessarily have to exist but it must be valid.
```

Comment:
it must be valid may lead to many interpretations. Maybe you can replace it by it must follow the email's standardization and attach a link that explain it...

Reply:
I updated the description of the variables.